geocoding_ban/custom-recipes/geocoding_ban/recipe.json

```diff
@@ -0,0 +1,98 @@
+{
+    // Meta data for display purposes
+    "meta" : {
+        "label" : "Geocode with BAN",
+        "author" : "Ministère de l’intérieur — EIG 2017",
+        "tags" : ["geocoding", "dev"]
+    },
+
+    "kind" : "PYTHON",
+
+    "inputRoles" : [{
+        "name": "input",
+        "label": "Dataset to geocode",
+        "description": "what input A means",
+        "arity": "UNARY",
+        "required": true,
+        "acceptsDataset": true
+    }],
+    "outputRoles" : [{
+        "name": "output",
+        "label": "Output dataset",
+        "description": "what main output means",
+        "arity": "UNARY",
+        "required": true,
+        "acceptsDataset": true
+    }],
+
+    "params": [
+        {
+            "name": "server_address",
+            "label" : "Server hosting Addok",
+            "type": "STRING",
+            "description":"Full url with http://. Do not include the path /search",
+            "mandatory" : true,
+            "defaultValue" : "http://api-adresse.data.gouv.fr"
+        },
+        {
+            "name": "columns",
+            "label" : "Address columns",
+            "type": "COLUMNS",
+            "description":"Multiple columns will be concatenated",
+            "mandatory" : true
+        },
+        {
+            "name": "post_code",
+            "label" : "Column of the postcode",
+            "type": "COLUMN",
+            "description":"Only results of that postcode will be returned.",
+            "mandatory" : false
+        },
+        {
+            "name": "city_code",
+            "label" : "Column of the city code",
+            "type": "COLUMN",
+            "description": "Only results of that city code will be return. Do not confuse it with postcodes (despite its ressemblance in France)",
+            "mandatory" : false
+        },
+        {
+            "name": "lines_per_request",
+            "label": "Lines per request",
+            "type": "INT",
+            "defaultValue": 1000,
+            "mandatory": true,
+            "description": "Sending multiple requests in one iteration saves time"
```

Comment:
Since the name is "Lines per request", we might want to avoid "requests per iteration", and repeat line per request in the description. Since the API limits to 8MB (and since I guess returns are diminishing), it might be useful to state that you should generally not go above 1000